### PR TITLE
add dipper

### PR DESCRIPTION
**Pull request by zhullyb**

### CODE REVIEW

File: dipper.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "buildtype": "Monthly",
+        "forum": "https://forum.xda-developers.com/t/rom-official-12-0-crdroid-dipper.4398843/",
+        "gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-S/",
+        "firmware": "https://xiaomifirmwareupdater.com/firmware/dipper/",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://dl.twrp.me/dipper/",
+        "paypal": "",
+        "telegram": "https://t.me/zhullyb_release",
+        "dt": "https://github.com/Bamice/android_device_xiaomi_dipper",
```
Comment (gwolf2u) [inline dipper.json]: save dt to crDroid github as per rules
https://github.com/crdroidandroid/rules-and-guidelines#general-rules

File: dipper.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "common-dt": "https://github.com/Bamice/android_device_xiaomi_sdm845-common",
```
Comment (gwolf2u) [inline dipper.json]: this one also

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by zhullyb** [comment]
Done.

**Review by gwolf2u — APPROVED** [review]

File: dipper.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "竹林里有冰 (zhullyb)",
+        "oem": "Xiaomi",
+        "device": "Xiaomi MI 8",
+        "filename": "crDroidAndroid-12.0-20220206-dipper-v8.1.zip",
+        "download": "https://sourceforge.net/projects/zhullyb/files/dipper/12.0/crDroid/crDroidAndroid-12.0-20220206-dipper-v8.1.zip/download",
```
Comment (gwolf2u) [inline dipper.json]: correct d/l url

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: dipper.json (hunk 3 above)
Comment (zhullyb) [reply dipper.json]: Done, please review again

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**